scripts/generate-faq-images.R

```diff
@@ -0,0 +1,129 @@
+library(magrittr)
```

Comment:
Proposing an alternative: make this in `alsf-scpca` and then use a permalink to the PNG in the docs themselves. May also consider if we should use the link to `main`, rather than a specific commit if things change. I could see an argument for both ways.

Reply:
Also going to add: Should we be using an `SCPCL` identifier in these plots to minimize confusion?